test/all/index.js

```diff
@@ -208,6 +208,15 @@ test('makePublic', function(t) {
   })
 });
 
+test('updateBucket', function(t) {
```

Comment:
This shadows the `makePublic` test. Line 220 tests `makePublic` worked by downloading the file using an unauthenticated client. `makePublic` could be broken and this test would 'fix' the remote state by adding `PULL` back to the permissions before we validated we could download the file.

I'm not sure how much we can update on a bucket, but could we change something other than permissions?

Also, we need to not only use `updateBucketById`, but we also need to use `getBucket` after to verify subsequent calls reflect that state.